index.html

```diff
@@ -811,99 +807,97 @@ <h3>
-                <a href=
-                "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-fetchstart">
+                <a data-cite=
+                "resource-timing-2#dom-performanceresourcetiming-fetchstart">
                 <code>fetchStart</code></a>.
               </li>
-              <li>Let <a href=
-              "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-redirectend">
-                <code>redirectEnd</code></a> be the value of <a href=
-                "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-responseend">
+              <li>Let <a data-cite=
+              "resource-timing-2#dom-performanceresourcetiming-redirectend">
+                <code>redirectEnd</code></a> be the value of <a data-cite=
+                "resource-timing-2#dom-performanceresourcetiming-responseend">
                 <code>responseEnd</code></a>.
               </li>
               <li>Set all of the attributes in the
               <a>PerformanceNavigationTiming</a> object to 0 except
-              <a>startTime</a>, <a href=
-              "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-redirectstart">
-                <code>redirectStart</code></a>, <a href=
-                "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-redirectend">
+              <a>startTime</a>, <a data-cite=
+              "resource-timing-2#dom-performanceresourcetiming-redirectstart">
+                <code>redirectStart</code></a>, <a data-cite=
+                "resource-timing-2#dom-performanceresourcetiming-redirectend">
                 <code>redirectEnd</code></a>, <a data-link-for=
                 "PerformanceNavigationTiming">redirectCount</a>,
                 <a data-link-for="PerformanceNavigationTiming">type</a>,
-                <a href=
-                "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-nexthopprotocol">
+                <a data-cite=
+                "resource-timing-2#dom-performanceresourcetiming-nexthopprotocol">
                 <code>nextHopProtocol</code></a>, <a data-link-for=
                 "PerformanceNavigationTiming">unloadEventStart</a> and
                 <a data-link-for=
-                "PerformanceNavigationTiming">unloadEventEnd</a>. Set <a href=
-                "https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-nexthopprotocol">
+                "PerformanceNavigationTiming">unloadEventEnd</a>. Set <a data-cite=
+                "resource-timing-2#dom-performanceresourcetiming-nexthopprotocol">
                 <code>nextHopProtocol</code></a> to the empty
-                <code>DOMString</code>.
+                <code><a>DOMString</a></code>.
               </li>
-              <li>Return to <a href="#fetch-start-step">fetch-start-step</a>
+              <li>Return to <a data-cite="#fetch-start-step">fetch-start-step</a>
```

Comment:
Nit: just `<a>fetch-start-step</a>` ? It's in the same document.

Reply:
oh, I see why. It's defined as a `id="fetch-start-step"` on the `li`... that's a little bit unconventional,  but maybe just revert this one to `<a href="#fetch-start-step">`. That should be fine.